### DISCUSSION TITLE
Questions related to RA & TA

### DISCUSSION THREAD

**Discussion by systemnerd** [snigi-gupta/AttendingUniversityAtBuffalo#38] [Q&A] [answered]
Hello Snigdha, here are my questions:

1. Is RA & TA strictly reserved for PhD students only? Or MS students also get it.
2. Does opting for thesis track increase our chances of getting RA/TA?
3. How & when should we start approaching the profs for this position? Can you guide someone who does not have previous teaching experience?
4. Will there be a reduction in the tuition fee if we manage to secure the position OR it's going to be a pay per hour basis?

**Comment by snigi-gupta** [comment, marked as answer]
Hi @NithinSastry,

1. There is no such rule. This is just an observation that since Ph.D. students stay longer at the university, it makes sense to hire them as TA/RA. But having said that, there is no such rule or compulsion. If you are eligible for some job/role, I highly recommend active applying.
2. No, there is no such thing that taking a thesis will grant you RA/TA sooner. It is only and solely dependent on the professor. I know a few MS students without a thesis who got RA/TA.
3. I would recommend that when it comes to university jobs, always start applying 1 sem in advance. These are always high in demand and get filled quickly. Try to network with your seniors and seek help from them. If a senior is already working as an RA, then getting a referral from them can be a huge help.
4. Lookout for internships. They pay a lot more than RA/TA and hence can also be a good source of income. Also, check for a GA role on handshake. I knew someone in my batch who got a GA, and that means their tuition was cut by 50%.

Feel free to post more questions, I am now answering questions every Sunday!
